scripts/apps/search/MultiImageEdit.ts

```diff
@@ -120,7 +120,19 @@ export function MultiImageEditController(
 
         $scope.images.forEach((item) => {
             if (item.selected) {
-                item[field] = $scope.metadata[field] || '';
+                if (field === 'extra') {
+                    Object.keys(item[field]).forEach((key) => {
+                        if ($scope.metadata[field][key]) {
+                            item[field][key] = $scope.metadata[field][key];
+                        } else {
+                            delete item[field][key];
```

Comment:
Instead of deleting in here, could it instead be prevented from being added in the first place?